crates/sui-core/src/authority/authority_store.rs

```diff
@@ -219,7 +220,28 @@ impl AuthorityStore {
             .contains_key(digest)?)
     }
 
-    pub fn insert_executed_transactions(
+    pub fn get_transaction_cert_sig(
+        &self,
+        tx_digest: &TransactionDigest,
+    ) -> Result<Option<AuthorityStrongQuorumSignInfo>, TypedStoreError> {
+        self.perpetual_tables
+            .transction_cert_signatures
+            .get(tx_digest)
+    }
+
+    // TODO: Move this to epoch store.
+    pub fn insert_transaction_cert_sig(
+        &self,
+        tx_digest: &TransactionDigest,
+        sig: &AuthorityStrongQuorumSignInfo,
+    ) -> SuiResult {
```

Comment:
It's interesting that in some DB requests (ie `insert`) we return `SuiResult` and on `get` we return `TypedStoreError`.

Reply:
Ideally they should all return TypedStoreError. I agree this can be cleaned up.